src/vitamin.h

```diff
@@ -61,8 +61,8 @@ class vitamin
         int min_;
         int max_;
         int rate_;
-        std::vector<std::pair<efftype_id, int>> deficiency_;
-        std::vector<std::pair<efftype_id, int>> excess_;
+        std::vector<std::pair<int, std::pair<efftype_id, int>>> deficiency_;
+        std::vector<std::pair<int, std::pair<efftype_id, int>>> excess_;
```

Comment:
These look very alike in every aspect. Could they be replaced with a single vector having values below/above zero to represent deficiency/excess respectively (as it is now)? This would prevent duplication.